go/private/rules/cgo.bzl

```diff
@@ -397,7 +397,7 @@ _cgo_collect_info = go_rule(
 """No-op rule that collects information from _cgo_codegen and cc_library
 info into a GoSourceList provider for easy consumption."""
 
-def setup_cgo_library(name, srcs, cdeps, copts, cxxopts, cppopts, clinkopts, objc, objcopts, tags):
+def setup_cgo_library(name, srcs, cdeps, copts, cxxopts, cppopts, clinkopts, objc, objcopts, **extra_args):
```

Comment:
One small nit: rename `extra_args` to `common_attrs`.

Reply:
Makes sense. Done.